src/main/webapp/app/config/constants.tsx

```diff
@@ -78,6 +78,8 @@ export const NOTIFICATION_TIMEOUT_MILLISECONDS = 5000;
 export const TOKEN_ABOUT_2_EXPIRE_NOTICE_IN_DAYS = 14;
 export const USAGE_TOP_USERS_LIMIT = 10000;
 export const USGAE_ALL_TIME_KEY = 'All';
+export const USGAE_ALL_TIME_VALUE = 'One year';
+export const USAGE_DETAIL_TIME_KEY = 'Detail';
```

Comment:
Should these two be updated?

Reply:
I remember we think 'One Year' looks good, and the 'Detail' is not for showing